brainstormforce/ultimate-addons-for-gutenberg
# Social share separation

### Description

Social share - Separation of an individual social share as a child block.

### Screenshots

https://abhijitn-bsf.tinytake.com/tt/NDIyODcxMl8xMzIxNTk5Ng

### Types of changes

Breaking change

### How has this been tested?

1. Create Social Share in the master branch.
2. Checkout to this branch and check is there any CSS change occur or not.

### Checklist:
